playground/menu.json

```diff
@@ -4,7 +4,27 @@
   "icon": "RocketIcon",
   "permissions": ["ManageDeveloperSettings", "ViewDeveloperSettings"],
   "featureToggle": null,
-  "submenu": [],
+  "submenu": [{
```

Comment:
This is needed for the VRTs?

Reply:
Ah sorry I was testing something...I forgot to remove it